src/components/ReservatioPage.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import SingleReservation from './SingleReservation';
+import Logo from './Logo.js';
+import Toolbar from './Toolbar';
+
+export default class ReservatioPage extends Component {
+    constructor(props) {
+        super(props)
+    
+        this.state = {
+             haircutDetails:[{ id:1,timezone:"11:00 - 11:20", status:"free",color:"green"},
+             {id:2,timezone:"11:20 - 11:40", status:"free",color:"green"},
+             {id:3,timezone:"11:40 - 12:00", status:"taken",color:"red"},
+             {id:4,timezone:"12:00 - 12:20", status:"free",color:"green"},
+             {id:5,timezone:"12:20 - 12:40", status:"cancel",color:"yellow"},
+             {id:6,timezone:"12:40 - 13:00", status:"free",color:"green"},
+             {id:7,timezone:"13:00 - 13:20", status:"free",color:"green"},
+             ]
+        }
+    }
+
+    updateResStatus=(status, condition,color)=>{
```

Comment:
The name of the function is not understand-able
Either any one of the parameters